crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1213,7 +1107,14 @@ impl<'a> SuiTestAdapter<'a> {
     // stable way of sorting objects by type. Does not however, produce a stable sorting
     // between objects of the same type
     fn get_object_sorting_key(&self, id: &ObjectID) -> String {
-        match &self.storage.get_object(id).unwrap().data {
```

Comment:
Writing a helper for `self.validator.database.get_object(id).unwrap().unwrap().data` might be worthwhile -- seems like quite a common operation.